hil-test/tests/gpio.rs

```diff
@@ -1,7 +1,7 @@
 //! GPIO Test
 
 //% CHIPS: esp32 esp32c2 esp32c3 esp32c6 esp32h2 esp32s2 esp32s3
-//% FEATURES: unstable embassy
+//% FEATURES(unstable): unstable embassy
 //% FEATURES(stable):
```

Comment:
According to the docs:
>  If you need to specify a feature of a dependency, you can use the `crate-name/feature-name` format.

But we dont have a `stable` feature (?)

Reply:
That's not a feature, it's the name of the configuration 🤔 let me see if I can explain this more clearly.